app/src/main/java/space/taran/arknavigator/ui/extra/VideoExtraLoader.kt

```diff
@@ -23,20 +23,20 @@ object VideoExtraLoader {
             )
         }
 
-        val duration = extra.data[MetaExtraTag.DURATION]
+        val duration = extra.data[MetaExtraTag.DURATION]?.toInt()
```

Comment:
`toIntOrNull()` instead of `toInt()` ?

Reply:
@darkovski88 this seems to not depend on user input, if we can't parse `Int` here, then we made error during saving.